src/Controls/src/Core/HandlerImpl/Toolbar/Toolbar.Windows.cs

```diff
@@ -46,7 +46,7 @@ internal void UpdateMenu()
 				{
 					button.SetBinding(AppBarButton.IconProperty, "IconImageSource", _imageSourceIconElementConverter);
 				}
-				else
+				else if (!item.IconImageSource.IsEmpty)
```

Comment:
Added this check, otherwise, it causes an NRE [here](https://github.com/dotnet/maui/blob/main/src/Controls/src/Core/Platform/Windows/ImageConverter.cs#L16)

Reply:
I'm still getting an NRE here because `IconImageSource` might be null

Reply:
I think there is a `ImageSource.IsNullOrEmpty()` that you can use...

Reply:
@mattleibow I found it [here](https://github.com/dotnet/maui/blob/main/src/Compatibility/Core/src/Tizen/Extensions/ImageExtensions.cs#L58)